test/integration/openshift_health_checker/preflight/playbooks/package_version_plain.yml

```diff
@@ -0,0 +1,21 @@
+---
+- include: ../../setup_container.yml
+  vars:
+    scenario: package_version_plain
+    image: openshift/test-target-base
+    l_host_vars:
+      deployment_type: origin
+      openshift_release: 1.4
+
+- name: Run package_version preflight check expecting to succeed
+  hosts: all
+  roles:
+    - openshift_health_checker
+  tasks:
+    - block:
+        - action: openshift_health_check
+          args:
+            checks: [ 'package_version' ]
+      # we want to destroy the container whether check passed or not
+      always:
+        - include: ../../teardown_container.yml
```

Comment:
Unfortunately blocks and exception handling only work for tasks... Cannot run a play, then recover from it and clean up in the next play.

There isn't a way, as far as I know, to pass in a task list to some kind of wrapper that does something before and after... although maybe we could create a plugin for that! As it is, yes, each test playbook is going to have some copy-paste, I was trying to at least minimize it.